src/assets/i18n/en.json

```diff
@@ -18,6 +18,34 @@
     "ANALYTICS_FALLBACK": "Some analytics here in the future",
     "SIDENAV": {
       "SLOTS": "SLOTS"
+    },
+    "CREATE_TIME_SLOT_CARD": {
```

Comment:
remove `_CARD`

Reply:
fixed in https://github.com/aleks00onyshko/touche-adminstration-FE/pull/6/commits/65576347a418f94dd3396e1348ffce56e9fea0f1